SpeechResponder/VariablesWindow.xaml.cs

```diff
@@ -33,7 +33,7 @@ public VariablesWindow(string scriptName)
             try
             {
                 DirectoryInfo dir = new DirectoryInfo(System.IO.Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
-                markdown = Files.Read(dir.FullName + @"\Variables.md");
+                markdown = File.ReadAllText(dir.FullName + @"\Variables.md");
```

Comment:
This change is not necessary. Files.Read() utilizes File.ReadAllText().

Reply:
i try with a french translation of Help.md and, the change is real. Perhabs it have no change for you because english don't use
Before change : 

> Scripter avec EDDI
> Le processeur vocal de EDDI utilise Cottle pour les scripts. Cottle a un grand nombre de possibilitÃ©s dont:
> 
> * CapacitÃ© de dÃ©finir et de mettre Ã  jour des variables, y compris des tableaux
> * Gestion de boucles
> * tests conditionnels
> * Liaisons entre scripts

and after the change:

> Scripter avec EDDI
> Le processeur vocal de EDDI utilise Cottle pour les scripts. Cottle a un grand nombre de possibilités dont:
> 
> * Capacité de définir et de mettre à jour des variables, y compris des tableaux
> * Gestion de boucles
> * tests conditionnels
> * Liaisons entre scripts

Reply:
What is the encoding on the French version of Help.md? I suspect the French translation was saved with encoding for 'UTF-8' while the English version was not. If so, then I think that saving the .md files in UTF-8 encoding is what really drives the change.